src/main/java/org/folio/edge/sip2/repositories/PatronRepository.java

```diff
@@ -40,21 +44,30 @@
  *
  */
 public class PatronRepository {
+  private static final String FIELD_REQUESTS = "requests";
   private static final String FIELD_TITLE = "title";
   private static final String FIELD_TOTAL_RECORDS = "totalRecords";
   private static final Logger log = LogManager.getLogger();
+  // These really should come from FOLIO
+  static final String MESSAGE_INVALID_PATRON =
+      "Your library card number cannot be located.  Please see a staff member for assistance.";
```

Comment:
Can this message be more generic? "Library card number" sounds specific. I get that if it can't be located it is a reason for the patron to be invalid.  I can't think of an alternative message to suggest, however.

Reply:
I agree they are not that great, but these are the messages Chalmers receives now. It was suggested to use these until [UXPROD-266](https://issues.folio.org/browse/UXPROD-266) is implemented.

Reply:
How did you know about this story. Good job being aware of the relevant parallel discussions.

Reply:
Thank our illustrious PO. She pointed me to it. 😄